RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -351,6 +358,26 @@ int PixelThresholdClusterizer::calibrate(int adc, int col, int row)
 	  } else {
 	    electrons = int( vcal * theConversionFactor + theOffset); 
 	  }
+
+
+	  // if(adc<0 || adc>255) cout<<" adc wrong range "<<adc<<endl;
```

Comment:
Please remove unneded commented out code, or add a comment which explains why it is needed here